samples/msal-browser-samples/TestingSample/test/browser-test.spec.ts

```diff
@@ -1,56 +1,57 @@
-import "expect-puppeteer";
-import * as msal from "@azure/msal-node";
-
-// Credentials
-const [username, password] = getCredentials(); // Implement an API to get your credentials
+import { test, expect, type Page } from "@playwright/test";
+import { PublicClientApplication, CacheKVStore } from "@azure/msal-node";
 
 const config = {
-  auth: {
-    clientId: "client_id_here", // Add same ClientId here as in app/authConfig.js
-    authority: "add_tenanted_authority_here" // Add same tenanted authority here as in app/authConfig.js
-  } 
+    auth: {
+        clientId: "Enter_Your_Client_Id_Here", // Add same ClientId here as in app/authConfig.js
+        authority:
+            "https://login.microsoftonline.com/Enter_the_Tenant_Id_Here", // Add same tenanted authority here as in app/authConfig.js
+    },
 };
 
-// Defining the timeout for the test
-const timeout = 8000;
-let tokenCache: msal.CacheKVStore;
-
-async function setSessionStorage(tokens: msal.CacheKVStore) {
-  const cacheKeys = Object.keys(tokenCache);
-  for (let key of cacheKeys) {
-    const value = JSON.stringify(tokenCache[key]);
-    await page.evaluate((key, value) => {
-      sessionStorage.setItem(key, value);
-    }, key, value);
-  };
-  await page.reload();
+let tokenCache: CacheKVStore;
+
```

Comment:
Can we pass a link to https://playwright.dev/docs/auth for alternative ways of populating storage with auth data

Reply:
Updated